client/src/components/grants/inputs.tsx

```diff
@@ -43,20 +46,21 @@ export function TextInputAddress({
   changeHandler,
   displayError, // a string that will be: (visible | invisible)
 }: AddressInputProps) {
-  // todo: validate address
-
   return (
-    <div className="mt-6 w-full sm:w-1/2 static">
+    <div className="relative mt-6 w-full sm:w-1/2">
       <label className="text-sm absolute" htmlFor={name}>
         {label}
       </label>
-      <Tooltip bg="purple.600" hasArrow label={tooltipValue}>
+      <span className="absolute mr-8 text-purple-700 inset-y-0 right-0">
+        *Required
+      </span>
```

Comment:
Should this also be conditionally displayed based on `required`?

I guess we are assuming this will always be required, but it might be better to keep it consistent with the rest of the components.

Reply:
Agreed